sdk/src/client/bindings/python/iota_client/types/feature.py

```diff
@@ -0,0 +1,93 @@
+from enum import Enum
+
+
+class FeatureType(Enum):
+    Sender = 0
+    Issuer = 1
+    Metadata = 2
+    Tag = 3
+
+
+class Feature():
+    def __init__(self, type, sender=None, issuer=None, data=None, tag=None):
+        """Initialise a feature
+
+        Parameters
+        ----------
+        type : FeatureType
+            The type of feature
+        sender : Address
+            Sender address
+        issuer : Address
+            Issuer Address
+        data : string
+            Hex encoded metadata
+        tag : string
+            Hex encoded tag used to index the output
+        """
+        self.type = type
+        self.sender = sender
+        self.issuer = issuer
+        self.data = data
+        self.tag = tag
+
+    def as_dict(self):
+        config = {k: v for k, v in self.__dict__.items() if v != None}
+
+        config['type'] = config['type'].value
+
+        if 'sender' in config:
+            config['address'] = config.pop('sender').as_dict()
+
+        if 'issuer' in config:
+            config['address'] = config.pop('issuer').as_dict()
+
+        return config
+
+
+class SenderFeature(Feature):
+    def __init__(self, sender):
+        """Initialise a SenderFeature
```

Comment:
There's a mix of British and American English spelling in the docs. PLS ALIGN! :trollface:

Reply:
Done